x-pack/auditbeat/magefile.go

```diff
@@ -253,7 +253,7 @@ func installDependencies(pkg, arch string) error {
 		}
 	}
 
-	if err := sh.Run("apt-get", "update"); err != nil {
+	if err := sh.Run("apt-get", "update", "-o", "Acquire::Check-Valid-Until=false"); err != nil {
```

Comment:
Let's put a "TODO" in here as well saying that this is only for debian 7 and should be removed when move to a newer OS. This flag is going to be used unnecessary when building using non-debian7 images (like when making the linux/arm binaries) and we should remove it soonish.